src/main/groovy/cz/alenkacz/gradle/scalafmt/ScalafmtFormatBase.groovy

```diff
@@ -14,7 +15,13 @@ class ScalafmtFormatBase extends DefaultTask {
     def formatter = Scalafmt.create(cl)
             .withRespectVersion(false)
             .withDefaultVersion("1.5.1")
+            .withMavenRepositories(repositories())
 
+    private String[] repositories() {
+        project.getRepositories().stream().map { repository ->
```

Comment:
Are you sure that the `project` field is already set at this stage? (when initializing a field)

Reply:
Thanks, I'm new to gradle. Should I move the initialization of scalafmt to the run method then? Or do you have something else on your mind?

Reply:
i think moving this to method invocation is much safer, good point augi

Reply:
I don't exactly know that the builder exactly does, but creating them in a method (that is called when needed) seems to be OK for me 👍